skymap_scanner/config.py

```diff
@@ -74,10 +75,10 @@
 STATEDICT_GCDQP_PACKET: Final = "GCDQp_packet"
 STATEDICT_BASELINE_GCD_FILE: Final = "baseline_GCD_file"
 STATEDICT_NSIDES: Final = "nsides"
-STATEDICT_INPUT_PULSES: Final = "input_pulses_name"
 #
 MSG_KEY_RECO_ALGO: Final = "reco_algo"
 MSG_KEY_PFRAME_PKL_B64: Final = "pframe_pkl_b64"
+MSG_KEY_FORMAT_VERSION: Final = "format_version"
```

Comment:
I recommend renaming this key `realtime_format_version` and the variable `MSG_KEY_REALTIME_FORMAT_VERSION` to match the other use cases. This way, if we add another "version" variable in the future, there'd be no confusion (or refactoring needed).

Reply:
I think you missed the value--I suggest renaming the key in the message objects/ json files.